### PR TITLE
Bib 691 admin 404s should redirect to dashboard

### PR DESCRIPTION
**Pull request by carlosmorales125**

### CODE REVIEW

File: src/lib/logger.ts [hunk 1]
```diff
@@ -66,7 +66,13 @@ export const log = {
...
             console.error(error);
 
-            if (logToAppInsights) {
+            let statusCode: number | undefined;
+
+            if ('status' in error) {
+                statusCode = error.status;
+            }
+
+            if (logToAppInsights && statusCode !== 404) {
```
Comment (MrGrinst) [inline src/lib/logger.ts:75]: Could we use `isApiErrorWithStatus` here instead? https://github.com/BiblioNexusStudio/content-manager-web/blob/master//Users/kylegrinstead/Developer/content-manager-web/src/lib/utils/http-errors.ts#L40

**Review by MrGrinst — COMMENTED** [review]
@carlosmorales125 are we handling non-matching routes as well? For instance `/projectss` should redirect to dashboard.

File: src/routes/+error.svelte [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+    import { goto } from '$app/navigation';
+    import CenteredSpinnerFullScreen from '$lib/components/CenteredSpinnerFullScreen.svelte';
+
+    onMount(() => {
+        goto('/');
+    });
 </script>
 
-<h1>{$page.error?.message}</h1>
+<CenteredSpinnerFullScreen />
```
[the remark this answers is not in the document]
Comment (carlosmorales125) [reply src/routes/+error.svelte:11]: It's doable, but it's outside the scope of the ticket.

[the remark this answers is not in the document]
Comment (MrGrinst) [reply src/routes/+error.svelte:11]: I guess I'd argue that with this change the ticket is changing existing functionality... Could we do another follow-up ticket to address?

**Review by MrGrinst — APPROVED** [review]

**Pull request merged**